test/verify/check-terminal

```diff
@@ -99,6 +99,13 @@ PROMPT_COMMAND='printf "\\033]0;%s@%s:%s\\007" "${USER}" "${HOSTNAME%%.*}" "${PW
         wait_line(n + 1, blank_state)
         self.assertNotIn('admin', b.text(line_sel(n + 1)))
 
+        # Check that when we `exit` we can still reconnect with the 'Reset' button
+        b.key_press("exit\r")
+        b.wait_in_text(".terminal .xterm-accessibility-tree", "disconnected")
+        b.click('button:contains("Reset")')
+        wait_line(n, prompt)
```

Comment:
Does this line wait until the `PS1` thing appears? If so this is goodo

Reply:
Indeed, it does. It waits for the first line to contain __only__ prompt, thus it must have been reset otherwise it would also contain the word `exit`. Maybe would make a bit more sense to switch order of this line with the following one, but functionally it does not make any difference.